examples/tf/rl2_halfcheetah.py

```diff
@@ -0,0 +1,85 @@
+"""Example script to run RL2 in HalfCheetah."""
+from garage import wrap_experiment
+from garage.envs.half_cheetah_vel_env import HalfCheetahVelEnv
+from garage.experiment import task_sampler
+from garage.experiment.deterministic import set_seed
+from garage.np.baselines import LinearFeatureBaseline
+from garage.sampler import LocalSampler
+from garage.sampler.rl2_worker import RL2Worker
+from garage.tf.algos import RL2
+from garage.tf.algos import RL2PPO
+from garage.tf.algos.rl2 import RL2Env
+from garage.tf.experiment import LocalTFRunner
+from garage.tf.policies import GaussianGRUPolicy
+
+
+@wrap_experiment
+def rl2_ppo_halfcheetah(ctxt=None, seed=1):
+    """Train PPO with HalfCheetah environment.
+
+    Args:
+        ctxt (garage.experiment.ExperimentContext): The experiment
+            configuration used by LocalRunner to create the snapshotter.
+        seed (int): Used to seed the random number generator to produce
+            determinism.
+
+    """
+    set_seed(seed)
+    with LocalTFRunner(snapshot_config=ctxt) as runner:
+        max_path_length = 100
+        meta_batch_size = 10
+        n_epochs = 50
+        episode_per_task = 4
+
+        # ---- For ML1-push
+        # env = RL2Env(ML1.get_train_tasks('push-v1'))
+        # tasks = task_sampler.EnvPoolSampler([env])
+        # tasks.grow_pool(meta_batch_size)
+
+        # ---- For HalfCheetahVel
+        tasks = task_sampler.SetTaskSampler(lambda: RL2Env(
+            env=HalfCheetahVelEnv()))
+
+        env_spec = tasks.sample(1)[0]().spec
+        policy = GaussianGRUPolicy(name='policy',
+                                   hidden_dim=64,
+                                   env_spec=env_spec,
+                                   state_include_action=False)
+
+        baseline = LinearFeatureBaseline(env_spec=env_spec)
+
+        inner_algo = RL2PPO(
```

Comment:
this is a really strange API.

why not just have RL2PPO and RL2TRPO, which handle composing these objects automatically?

@krzentner PTAL